core/templates/dev/head/profile/profile.html

```diff
@@ -72,9 +72,7 @@ <h2 class="oppia-profile-username-small-screen">
         Interests:
         <span ng-if="subjectInterests.length > 0"><br>
           <span ng-repeat="interest in subjectInterests track by $index">
-            <span class="oppia-profile-subject-interest">
-              <[interest]>
-            </span>
+            <span class="oppia-profile-subject-interest"><[interest]></span>
```

Comment:
Could you please add an HTML comment above this line, along the following lines?

```
  <!-- The following tag should remain on a single line due to http://bugzilla.mozilla.org/show_bug.cgi?id=922773. -->
```

This helps to prevent future accidental regressions by developers. Thanks!